packaging/dbscripts/upgrade/04_05_0180_mdevs_as_vmdevices.sql

```diff
@@ -0,0 +1,45 @@
+CREATE FUNCTION tmp_custom_properties_to_vm_devices() RETURNS void AS $$
+DECLARE
+    vm record;
+    property record;
+    mtype record;
+    mdev_types text;
+    nodisplay boolean;
+    device_id uuid;
+    spec_params text;
+    new_properties text;
+BEGIN
+    FOR vm IN SELECT vm_guid, predefined_properties FROM vm_static
+                     WHERE predefined_properties LIKE '%mdev_type=%'
+    LOOP
+        FOR property IN SELECT * FROM regexp_split_to_table(vm.predefined_properties, ';')
```

Comment:
please use "AS" and give a meaningful name to the result 
for example ;
FROM regexp_split_to_table(vm.predefined_properties, ';') AS property where property LIKE 'mdev_type=%'

Reply:
Took a bit effort to find meaningful non-conflicting names, but done.